.cursor/hooks/post-edit-checks.sh

```
@@ -0,0 +1,119 @@
+
+MAX_RETRIES=5
+PROJECT_HASH=$(echo "$PWD" | md5sum | cut -c1-12)
+RETRY_FILE="/tmp/konflux-ui-post-edit-retries-${PROJECT_HASH}"
+
+# Detect platform: CURSOR_VERSION is always set in Cursor hooks, never in Claude Code
+is_cursor=false
+if [[ -n "${CURSOR_VERSION:-}" ]]; then
+  is_cursor=true
+fi
+
+hanged_files=$(git diff --name-only 2>/dev/null | grep -E '\.(ts|tsx)$' || true)
```

Shellcheck (0.11.0)

[warning] hanged_files appears unused. Verify use (or export if used externally).

(SC2034)